src/test/java/com/google/cloud/spanner/pgadapter/python/django/DjangoTestSetup.java

```diff
@@ -61,4 +61,26 @@ public String executeBasicTests(int port, String host, List<String> options)
 
     return output.toString();
   }
+
+  public String executeTransactionTests(int port, String host, List<String> options)
```

Comment:
nit: Could we combine this method with the one above. As far as I can see, the only difference between the two is the Python script file that we are calling.